.bazelproject

```diff
@@ -10,8 +10,7 @@ directories:
 targets:
   //java/...
   //javatests/...
-  //particles/Native/Wasm/...
-  //particles/Tutorial/Kotlin/...
+  //particles/...
```

Comment:
Is this also required/needed, or just an improvement to make sure we build all particles without having to update the paths here?

Reply:
Blackjack was added under `particles/`. My choices were to add that specific subfolder or target all particles. I figured as more kotlin particles are developed in the `particles/` path, we would like to avoid having to remember to update the bazelproject.